pkg/spanconfig/spanconfigtestutils/utils.go

```diff
@@ -110,10 +146,20 @@ func ParseKVAccessorGetArguments(t *testing.T, input string) []spanconfig.Target
 		}
 
 		const spanPrefix = "span "
-		if !strings.HasPrefix(line, spanPrefix) {
-			t.Fatalf("malformed line %q, expected to find spanPrefix %q", line, spanPrefix)
+		const systemTargetPrefix = "system-target "
+		switch {
+		case strings.HasPrefix(line, spanPrefix):
+			line = strings.TrimPrefix(line, spanPrefix)
+		case strings.HasPrefix(line, systemTargetPrefix):
+			line = strings.TrimPrefix(line, systemTargetPrefix)
+		default:
+			t.Fatalf(
+				"malformed line %q, expected to find spanPrefix %q or system target prefix %q",
```

Comment:
[nit] "expected to find %q or %q prefix" would avoid stuttering in your error message.